### PR TITLE
Fix Android app notification localization issue

### PR DESCRIPTION
**Pull request by wesley-firewalla**

### CODE REVIEW

File: bin/real [hunk 1]
```diff
@@ -1 +0,0 @@
-real.armv7l
```
Comment (MelvinTo) [inline bin/real:1]: why delete this?

File: controllers/netbot.js [hunk 2]
```diff
@@ -497,9 +497,9 @@ class netBot extends ControllerBot {
         notifMsg["title-loc-args"] = newArray;
         notifMsg["loc-key"] = alarm.localizedNotificationContentKey();
         notifMsg["loc-args"] = alarm.localizedNotificationContentArray();
-        notifMsg["title_loc_key"] = alarm.localizedNotificationTitleKey();
+        notifMsg["title_loc_key"] = alarm.localizedNotificationTitleKey().replace(/[.:]/g, '_');
```
Comment (MelvinTo) [inline controllers/netbot.js:500]: is this absolutely necessary? if changing box side, the localization support for android will be delayed for at least three months.

File: bin/real (hunk 1 above)
Comment (wesley-firewalla) [reply bin/real]: Revert this changes for now. When got time, will test it on firewalla red.

File: controllers/netbot.js (hunk 2 above)
Comment (wesley-firewalla) [reply controllers/netbot.js]: The `-` and `:` can not be used in resource name in android. So we can only fix it in box currently.

**Pull request merged**